core/src/main/java/tc/oc/pgm/PGMPlugin.java

```diff
@@ -394,6 +384,17 @@ private void registerListeners() {
     registerEvents(new JoinLeaveAnnouncer(matchManager));
   }
 
+  private boolean loadInitialMaps() {
+    try {
+      mapLibrary.loadNewMaps(false).get(300, TimeUnit.SECONDS);
+    } catch (TimeoutException e) {
+      getLogger().warning("Loading all maps took >30s, other maps will keep loading async.");
```

Comment:
Possible oversight: is the new map load timeout supposed to be 5 minutes, or 30 seconds?

```suggestion
      getLogger().warning("Loading all maps took >5m, other maps will keep loading async.");
```

Reply:
the oversight is actually the 300, it was 30 and supposed to stay at 30, added a 0 for debugging purposes